src/fs_event_wrap.cc

```diff
@@ -34,25 +34,25 @@ class FSEventWrap: public HandleWrap {
   size_t self_size() const override { return sizeof(*this); }
 
  private:
+  static const encoding kDefaultEncoding = UTF8;
+
   FSEventWrap(Environment* env, Local<Object> object);
   virtual ~FSEventWrap() override;
 
   static void OnEvent(uv_fs_event_t* handle, const char* filename, int events,
     int status);
 
   uv_fs_event_t handle_;
-  bool initialized_;
-  enum encoding encoding_;
+  bool initialized_ = false;
+  enum encoding encoding_ = kDefaultEncoding;
 };
```

Comment:
Could have, yes, but I think the C++11 way of assigning at definition is clearer; no need to look up the constructor to see if and how it's assigned.